README.md

```diff
@@ -0,0 +1,135 @@
+ # Consensus protocol
+  Consensus protocol called Succinct Attestation (SA from hereon). SA is a permission-less Proof-of-Stake-based consensus mechanism with statistical finality guarantees. SA belongs to the committee-based PoS subcategory due to its use of committees to finalize a block for a given round.
+ 
+   SA is permission-less, meaning that any eligible (eligibility properties are defined below) Dusk Network protocol participant is authorized to join and participate in the consensus. An eligible participant is defined with the following properties:
+ 
+- A participant with a pre-configured amount of DUSK locked as a stake (a.k.a Provisioner)
+- A participant, whose stake has a maturity of at least 2 epochs (a.k.a Eligible Provisioner)
+
+Other terms used below:
+- `Round` - A single SA execution
+- `Round iteration` - All three phases (Selection, 1th Reduction and 2th Reduction) executed in a row.
+- `Committee` - A subset of Eligible Provisioners, extracted on performing `Deterministic Sortition`
+
+# Repository structure
+
+## Example Node
+A minimalistic and stateless version of dusk-blockchain node where Consensus protocol in conjuction with Kadcast is able to join and participate `dusk-blockchain`/test-harness. Useful for testing and diagnostic incompatibility issues. This executable should be deprecated once dusk-blockchain is fully migrated.
+
+## Example Testbed
+A multi-instance setup running 10 SA instances provisioned with 10 eligible participants. The setup is configured to run for up to 1000 rounds. Useful for any kind of testing (issues, stress and performance testing).
+
+## Consensus crate
+A full implementation of SA mechanism.
+
+ # Implementation details 
+SA is driven basically by two tokio-rs tasks - Main Loop and Agreement Loop. Each of them has its own inbound and outbound message queues/channels to exchange messages with outsiders. SA protocol parameters are defined in `/src/config.rs`.
+
+- `Main_Loop uses` `Operations trait` to execute Contract Storage calls and `Database trait` to store and retrieve candidate blocks. It is mainly responsible to execute Selection, 1th Reduction and 2nd Reduction steps in a row and eventually produce/broadcast an **Agreement Message**. Inbound queue for Main_Loop can contain messages of either **NewBlock** or **Reduction** type.
```

Comment:
Like above, I find your uses of the inline code and bold text confusing. To me inline code is used to refer to code specific variables or types, like `NewBlock` or `Main_Loop` if they have the same name in the implementation. To put emphasis on text I would use *italic* or **bold** and for filepaths I always use single quotes 'srs/config.rs'.

Reply:
'1st Reduction'